react-example/src/Less/general.less

```diff
@@ -1,5 +1,5 @@
 * {
   box-sizing: border-box;
-  font-family: sans-serif;
-  line-height: 1.6rem;
+  font-family: 'Roboto', sans-serif;
+  font-size: 62.5%;
```

Comment:
This is kind of a hazard. Font `font-size`, what you really want to do is to set a `10px` value to the `html` element and then use `rem` units in the rest of the elements, so `1.4rem` will translate to `14px` with viewport scaling.